receiver/awsecscontainermetricsreceiver/README.md

```diff
@@ -3,21 +3,173 @@
+default: `20s`
+
+
+## Available Metrics
+Following is the full list of metrics emitted by this receiver.
+
+Task Level Metrics | Container Level Metrics | Unit 
+------------ | ------------- | --------------------
+ecs.task.memory.usage | container.memory.usage | Bytes
+ecs.task.memory.usage.max | container.memory.usage.max | Bytes
+ecs.task.memory.usage.limit | container.memory.usage.limit | Bytes
+ecs.task.memory.reserved | Bcontainer.memory.reserved | MB
+ecs.task.memory.utilized | container.memory.utilized | MB
+ecs.task.cpu.usage.total | container.cpu.usage.total | NS
+ecs.task.cpu.usage.kernelmode | container.cpu.usage.kernelmode | NS
+ecs.task.cpu.usage.usermode | container.cpu.usage.usermode | NS
+ecs.task.cpu.usage.system | container.cpu.usage.system | NS
+ecs.task.cpu.usage.vcpu | container.cpu.usage.vcpu | vCPU
+ecs.task.cpu.cores | container.cpu.cores | Count
+ecs.task.cpu.onlines | container.cpu.onlines | Count
+ecs.task.cpu.reserved | container.cpu.reserved | vCPU
+ecs.task.cpu.utilized | container.cpu.utilized | Percent
+ecs.task.network.rate.rx	| container.network.rate.rx	| Bytes/Sec
+ecs.task.network.rate.tx	| container.network.rate.tx	| Bytes/Sec
+ecs.task.network.io.usage.rx_bytes	| container.network.io.usage.rx_bytes	| Bytes
+ecs.task.network.io.usage.rx_packets	| container.network.io.usage.rx_packets	| Count
+ecs.task.network.io.usage.rx_errors |	container.network.io.usage.rx_errors	| Count
+ecs.task.network.io.usage.rx_dropped |	container.network.io.usage.rx_dropped	| Count
+ecs.task.network.io.usage.tx_bytes | container.network.io.usage.tx_bytes	| Bytes
+ecs.task.network.io.usage.tx_packets	| container.network.io.usage.tx_packets	| Count
+ecs.task.network.io.usage.tx_errors	| container.network.io.usage.tx_errors	| Count
+ecs.task.network.io.usage.tx_dropped	| container.network.io.usage.tx_dropped	| Count
+ecs.task.storage.read_bytes | container.storage.read_bytes| Bytes
+ecs.task.storage.write_bytes | container.storage.write_bytes | Bytes
+
+
+## Resource Attributes and Metrics Labels
+Metrics emitted by this receiver comes with a set of resource attributes. These resource attributes can be converted to metrics labels using appropriate processors/exporters (See `Full Configuration Examples` section below). Finally, these metrics labels can be set as metrics dimensions while exporting to desired destinations. Check the following table to see available resource attributes for Task and Container level metrics. Container level metrics have three additional attributes than task level metrics.
+
+Resource Attributes for Task Level Metrics | Resource Attributes for Container Level Metrics
+------------ | -------------
+ecs.cluster | ecs.cluster
+ecs.task-definition-family | ecs.task-definition-family
+ecs.task-arn | ecs.task-arn
+ecs.task-id | ecs.task-id
+ecs.task-definition-version | ecs.task-definition-version
+ecs.service | ecs.service
+"" | `container.name`
+"" | `container.id`
+"" | `ecs.docker-name` 
```

Comment:
These rows look weird when I view them...


Its also odd to have this as a table, a list might make more sense.

Either way, I think the container attributes shouldn't have special formatting, and the empty strings look weird.

Reply:
Updated.